server/src/main/java/io/pravega/schemaregistry/storage/impl/group/InMemoryGroupTable.java

```diff
@@ -34,7 +34,7 @@
 public class InMemoryGroupTable implements GroupTable<Integer> {
     @GuardedBy("$lock")
     @Getter(AccessLevel.NONE)
-    private final Map<TableKey, Value<TableValue, Integer>> table = new HashMap<>();
+    final Map<TableKey, Value<TableValue, Integer>> table = new HashMap<>();
```

Comment:
dont make this non private. mark the getter with AccessLevel packagePrivate.
In fact create an explicit getter and annotate it with `@VisibleForTesting` annotation.